joblistwork.inc.php

```diff
@@ -550,9 +550,9 @@
     Public jlwsid; 
 
     //later put properties here
-    Public function __construct($jobcode,$cuttingtype, $processcode, $totalquantity, $jobOutputList) {
+    Public function __construct($jobcode) {
 
-        parent::__construct($jobcode, $cuttingtype, $processcode, $totalquantity, $jobOutputList);
+        parent::__construct($jobcodet);

```

Comment:
This must be removed as well

Reply:
have to remove the whole line of parent::__construct($jobcodet);